src/workflows-app/SubmissionConfig.test.js

```diff
@@ -1,8 +1,5 @@
-import '@testing-library/jest-dom';
```

Comment:
`@testing-library` doesn't need to be imported by each test. It's imported in the global test setup.